java/arcs/sdk/android/storage/service/StorageServiceConnection.kt

```diff
@@ -66,9 +78,30 @@ class DefaultStorageServiceBindingDelegate(
     override fun bindStorageService(
         conn: ServiceConnection,
         flags: Int,
-        options: ParcelableStoreOptions
+        options: ParcelableStoreOptions?
+    ): Boolean {
+        return context.bindService(StorageService.createBindIntent(context, options!!), conn, flags)
```

Comment:
Rather than `!!`, this is a good opportunity to be more helpful to the user:

```kotlin
val options = requireNotNull(options) {
    "ParcelableStoreOptions are required when binding to the StorageService from a ServiceStore"
}
```